schemas/witnet/witnet.proto

```diff
@@ -256,6 +256,11 @@ message CheckpointBeacon {
     Hash hash_prev_block = 2;
 }
 
+message CheckpointVRF {
+    fixed32 checkpoint = 1;
```

Comment:
Why not `uint32`?

Reply:
I just followed the same structure as `CheckpointBeacon`. Is there a reason to have a fixex32 in the beacon and not in the vrf?

Reply:
`unit32` uses variable length encoding, so smaller values take fewer bytes, but values greater than or equal to `2^28` use 5 bytes. `fixed32` always uses 4 bytes, and is slightly cheaper to serialize and deserialize.

Reply:
Everyone here knows I'm a big fan of variable length encoding, but I understand @tmpolaczyk's rationale and in this case I do believe the right option is `fixed32`.

Otherwise we would be doing "early chain optimization", i.e. making encoding more compact in the early days with a negative trade-off for the future, once the project has gone mainstream ant the protocol would actually get the most of smaller data structures. No thanks!